debian/Dockerfile

```diff
@@ -51,7 +51,7 @@ RUN set -ex \
     cloud-init \
     cpufrequtils \
     curl \
-    cryptsetup \
+    cryptsetup-bin \
```

Comment:
please add a test for binary existence here https://github.com/metal-stack/metal-images/blob/master/test/inputs/goss.yaml#L8